test/resizing.jl

```diff
@@ -144,5 +144,20 @@ end
             @test ndims(R) == 0
             @test !(R === A)
         end
+
+    @testset "Interpolation" begin
+        img = rand(16,16)
+        out = ImageTransformations.imresize(img,(128,128),method=Lanczos4OpenCV())
+        @test size(out) == (128,128)
+
+        out = ImageTransformations.imresize(img,(128,128),method=Linear())
+        @test size(out) == (128,128)
+
+        out = ImageTransformations.imresize(img,(16,16),method=Lanczos4OpenCV())
+        @test size(out) == (16,16)
+
+        @test_throws ArgumentError imresize(img, BSpline(Linear()))
+        @test_throws ArgumentError imresize(img, Linear())
+        end
```

Comment:
Let's also add the following test cases with different `method` choices.

- offset array and make sure the offset is correctly handled, e.g., `imresize(img, (128, 128)) == imresize(OffsetArray(img, -1, -1), (128, 128))`
- axes: `imresize(img, (0:127, 0:127))` and make sure the `axes(out) == (0:127, 0:127)` and `OffsetArrays.no_offset_view(out) == imresize(img, (128, 128))`

Reply:
Are you suggesting these for all 3 interpolation methods `Linear`, `BSpline(Linear())`, and `Lanczos4OpenCV()`?

Reply:
ok, done for imresize. I tried to copy the formats in the imrotate and warp sections for the tests. Let me know if there are more that should be done.